lib/eventify_pro/client.rb

```diff
@@ -44,6 +47,12 @@ def process_error(error)
       raise Error, 'Could not publish event'
     end
 
+    def log_error(message, method, params:)
+      message = "[EVENTIFY_PRO] Message: #{message}\nMethod: #{method} with params: #{params}" # rubocop:disable LineLength
+
+      logger.info(message)
```

Comment:
If users pass their logger to initializer, we don't know if that logger is responding to `info` method, we expect it to do so, but we can't be sure, so we should check if it responds to `info`.
And if it does, only then we should log message.